pkg/resource/template/template_funcs.go

```diff
@@ -33,6 +35,7 @@ func newFuncMap() map[string]interface{} {
 	m["fileExists"] = isFileExist
 	m["base64Encode"] = Base64Encode
 	m["base64Decode"] = Base64Decode
+	m["hash"] = hashNodeName
```

Comment:
Since the output format of this hash is a bit odd (an IP address), how about making the function name `hashToIPv4`?

Reply:
ack